src/v/redpanda/admin/api-doc/partition.json

```diff
@@ -59,6 +59,37 @@
                 }
             ]
         },
+        {
+            "path": "/v1/partitions/{namespace}/{topic}",
```

Comment:
The difference between `/v1/partitions/{namespace}/{topic}` and `/v1/partitions` is very subtle.

What do you thing of adding `/v1/partitions/unfiltered` to return all partitions? For a topic it could be `/v1/partitions/unfiltered/{namespace}/{topic}`

Reply:
Alternatively, can we fix `/v1/partitions` to return full partitions?

Reply:
i was thinking about changing semantics of `v1/partitions` to return all but i think we may do it in separate PR as this is unrelated to the current one.

Reply:
got it. what do think of adding a parameter to control local / global mode e.g.

`/v1/partitions/{namespace}/{topic}` & `/v1/partitions/{namespace}/{topic}?mode=local` to return partitions hosted on this node

`/v1/partitions/{namespace}/{topic}?mode=global` to return all partitions

it seems it reduces confusion and opens a way to add the parameter to `/v1/partitions` later to have similar experience & semantic

Reply:
maybe we can make this a part of a follow up PR ?